src/animations/animatedImage/index.js

```diff
@@ -56,21 +86,15 @@ export default class AnimatedImage extends BaseComponent {
   }
 
   render() {
-    const {containerStyle, imageStyle, imageSource, loader, testId} = this.props;
+    const {containerStyle, loader} = this.props;
     return (
-      <View style={containerStyle} testID={testId}>
-        <Animated.Image
-          style={[{opacity: this.state.opacity}, imageStyle]}
-          source={imageSource}
-          onLoad={this.onLoad}
-        />
-        {this.state.isLoading && loader &&
+      <View testID={this.testID} style={containerStyle}>
```

Comment:
We have too many views going on here. 
Try to see how we can reduce View Hierarchy here. 
Since Avatar appears in Lists it's important to reduce hierarchy here.

Reply:
@ArnonZ  Did you try going over the View hierarchy? is it possible to reduce it somehow?